src/main/java/mate/model/Driver.java

```diff
@@ -6,12 +6,40 @@
     private Long id;
```

Comment:
I think we should define `toString()` method for `Driver` to display info in exception messages